src/rpdk/core/contract/templates/transformation.template

```diff
@@ -0,0 +1,5 @@
+var jsonata = require('jsonata')
+var model = {{input_model}}
+var expression = jsonata("{{jsonata_expression}}");
```

Comment:
thinking about this a bit more do you think it just makes sense to make this a static nodejs file that is a function with two arguments input_model and jsonata_expression?